src/main/java/com/minecraftabnormals/endergetic/core/mixin/PlayerListMixin.java

```diff
@@ -8,20 +8,31 @@
 import net.minecraft.entity.EntityType;
 import net.minecraft.entity.player.ServerPlayerEntity;
 import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.nbt.CompressedStreamTools;
 import net.minecraft.nbt.ListNBT;
 import net.minecraft.network.NetworkManager;
+import net.minecraft.server.MinecraftServer;
 import net.minecraft.server.management.PlayerList;
+import net.minecraft.util.registry.DynamicRegistries;
 import net.minecraft.world.server.ServerWorld;
+import net.minecraft.world.storage.PlayerData;
 import net.minecraftforge.fml.network.PacketDistributor;
+import org.spongepowered.asm.mixin.Final;
 import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
+import java.io.File;
 import java.util.List;
 
 @Mixin(PlayerList.class)
 public final class PlayerListMixin {
+	@Shadow @Final
+	private PlayerData playerDataManager;
+	@Shadow @Final
+	private MinecraftServer server;
```

Comment:
We don't style our annotations this way.
```suggestion
	@Shadow
	@Final
	private PlayerData playerDataManager;
	@Shadow
	@Final
	private MinecraftServer server;
```